lighthouse/tests/test_lighthouse.py

```diff
@@ -58,6 +58,47 @@ def test_check(aggregator, instance):
     aggregator.assert_all_metrics_covered()
 
 
+@pytest.mark.integration
+def test_check_urls_tags(aggregator, instance):
+    common_tag = "key:value"
+    instance = {
+        "urls": ["https://www.google.com", "https://www.datadoghq.com"],
+        "name": "test",
+        "tags": [common_tag],
+    }
+    lighthouse_check = LighthouseCheck("lighthouse", {}, {})
+    LighthouseCheck._get_lighthouse_report = MagicMock(side_effect=mock_get_lighthouse_report)
+
+    lighthouse_check.check(instance)
+
+    expected_tags = [[common_tag, "url:" + url, "name:" + instance["name"]] for url in instance["urls"]]
```

Comment:
To make sure I understand the expected result, you want to see `tags=['key:value', 'name:test', 'name:test', 'url:https://www.datadoghq.com', 'url:https://www.google.com']` but not `tags=['key:value', 'name:test', 'name:test', 'url:https://www.datadoghq.com,url:https://www.google.com']`?

Reply:
The expected result is `tags=['key:value', 'name:test', 'url:https://www.datadoghq.com']` for metrics created when it runs the check on the DD url, and `tags=['key:value', 'name:test', 'url:https://www.google.com']` for metrics created when it runs the check on the Google url.

Hopefully the commit I pushed makes the expectation clearer.

---

The test failure I pasted in the PR description is what happens when I run this new test on the previous version of the check code, and demonstrates the undesired previous behaviour in the "similar submitted" metrics.

It created `tags=['key:value', 'name:test', 'url:https://www.google.com']` on the Google URL (the first in the list), but the DD URL (the second in the list) had metrics tagged with `tags=['key:value', 'name:test', 'name:test', 'url:https://www.datadoghq.com', 'url:https://www.google.com']`, which contains undesired tags from the previous URL in the list, the Google one.